src/coreclr/jit/instr.cpp

```diff
@@ -1886,6 +2001,15 @@ instruction CodeGenInterface::ins_Store(var_types dstType, bool aligned /*=false
         ins = INS_strb;
     else if (varTypeIsShort(dstType))
         ins = INS_strh;
+#elif defined(TARGET_LOONGARCH64)
+    if (varTypeIsByte(dstType))
+        ins = aligned ? INS_stx_b : INS_st_b;
+    else if (varTypeIsShort(dstType))
+        ins = aligned ? INS_stx_h : INS_st_h;
+    else if ((TYP_INT == dstType) || (TYP_UINT == dstType))
+        ins = aligned ? INS_stx_w : INS_st_w;
+    else // if ((TYP_LONG == dstType) || (TYP_ULONG == dstType) || (TYP_REF == dstType))
```

Comment:
Either add this `if` as an assert, or remove it.

(Presumably it could also be TYP_BYREF?)

Reply:
Yes.
Thanks, I will delete it.